Add info on how to include in projects (#89)
jellyfin/jellyfin-apiclient-java: README.md

This library allows Java and Android applications to easily access Jellyfin servers.
The dependencies are modular and can easily be swapped out with alternate implementations when desired.

## Setup

The apiclient is available through [JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and thus can be installed via Gradle like any other dependency:

```kotlin
// build.gradle.kts
repositories {
	jcenter()
}

dependencies {

> 	val apiclientVersion = "0.7.2"

Reviewer: ```suggestion
```
I think we shouldn't actually mention the version in this snippet so users won't end up using an older version if we don't update the README. The actual version to use is shown in the releases page and the badge on top of the readme.

Author: Right, updating the README is always a bit of a problem. Do we still keep the version variable, just with `…` in the string? I think that'd be the most obvious.

Reviewer: yes

	}
});
```

> ## Include into project

Reviewer: ```suggestion
## Install
```

Author: Don't know if "Install" is the right word, maybe "Setup"?

Reviewer: Setup sounds good!

> The Jellyfin api client is available through [Bintray/JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and can be included in projects like any other dependency:

Reviewer: ```suggestion
The apiclient is available through [JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and thus can be installed via Gradle like any other dependency:
```

```groovy
// build.gradle
dependencies {

> 	implementation 'org.jellyfin.apiclient:android:0.7.2'

Reviewer: Move the Groovy sample down (Kotlin-first) and rewrite with my suggestion

}
```

> ```kotlin

Reviewer: add `repositories { jcenter() }`

// build.gradle.kts
dependencies {

> 	implementation("org.jellyfin.apiclient:android:0.7.2")

Reviewer: ```suggestion
	val apiclientVersion = "0.7.2"
	
	// For non Android projects
	implementation("org.jellyfin.apiclient:library:$apiclientVersion")
	
	// For Android apps (automatically includes the library and models)
	implementation("org.jellyfin.apiclient:android:$apiclientVersion")
```

}

> ```

Reviewer: Move this section up above the code samples
